### ISSUE TITLE
MvxFragment error

### ISSUE THREAD

**Issue by Bosch571** [MvvmCross/MvvmCross#1030]
Hello everyone,

I'm getting the following error: "Trying to instantiate a class fyfe.mobileapps.droid.views.SubFrag that is not a Fragment". I've got no idea what i'm doing wrong :)

I have found a similar problem here https://github.com/MvvmCross/MvvmCross/issues/994 but I'm unsure what the solution or workaround is.

My activity:

```
[Activity(Label = "Start")]
public class StartView : MvxFragmentActivity
{
    public new StartViewModel ViewModel
    {
       get { return (StartViewModel)base.ViewModel; }
       set { base.ViewModel = value; }
    }

    protected override void OnViewModelSet()
    {
        SetContentView(Resource.Layout.MenuBase);

        var sub = (SubFrag)SupportFragmentManager.FindFragmentById(Resource.Id.contentFragment);
        sub.ViewModel = ViewModel.Sub;
    }
}
```
The fragment:
```
[Register("fyfe.mobileapps.droid.views.SubFrag")]
public class SubFrag : MvxFragment
{
    public override View OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle     savedInstanceState)
    {
        var ignored = base.OnCreateView(inflater, container, savedInstanceState);
        return this.BindingInflate(Resource.Layout.StartView, null);
    }
}
```
Fragment part of my .axml file
```
<fragment
          class="fyfe.mobileapps.droid.views.SubFrag"
          android:id="@+id/contentFragment"
          android:layout_width="fill_parent"
          android:layout_height="fill_parent" />
```

**Comment by martijn00** [comment]
Fixed in the next release

**Comment by ggirard-distech** [comment]
@martijn00 do you have a PR ref for this fix? Is this #1022 (originally reported by issue #1000)?
I am currently in the process to upgrade to 3.5.1 and I think I'm hitting the same issue. Thx.

**Comment by martijn00** [comment]
I try to remember, and i think it was #1022 indeed. I would advice you to update to 4.0.0 instead of 3.5.1.
